Sources/SwiftDocC/Semantics/Symbol/Relationship.swift

```diff
@@ -27,4 +27,6 @@ public enum Relationship {
     case defaultImplementation(TopicReference)
     /// A protocol of which the current node is a requirement.
     case requirementOf(TopicReference)
+    /// A protocol of which the current node is an optional requirement.
+    case optionalRequirementOf(TopicReference)
```

Comment:
Just a note, this breaks public API (if client switches over these cases without a `default`). It would be good to call this out in the PR description so that users are aware of this change.